### ISSUE TITLE
Tensorflow model working for sentence with exactly 5 words only

### ISSUE THREAD

**Issue by pmg1991** [deepjavalibrary/djl#1310]
## Description
Tensorflow model working for sentence with exactly 5 words only. Need help Am i doing anything wrong.

### Expected Behavior
(what's the expected behavior?)

### Error Message

```
Exception in thread "main" ai.djl.translate.TranslateException: org.tensorflow.exceptions.TFInvalidArgumentException: Incompatible shapes: [768,768] vs. [768,5,768]
	 [[{{node tf_distil_bert_for_sequence_classification/distilbert/embeddings/add/add}}]]
	at ai.djl.inference.Predictor.batchPredict(Predictor.java:179)
	at ai.djl.inference.Predictor.predict(Predictor.java:116)
	at com.pmg.nlp.BertClassification.predict(BertClassification.java:87)
	at com.pmg.nlp.BertClassification.main(BertClassification.java:48)
Caused by: org.tensorflow.exceptions.TFInvalidArgumentException: Incompatible shapes: [768,768] vs. [768,5,768]
	 [[{{node tf_distil_bert_for_sequence_classification/distilbert/embeddings/add/add}}]]
	at org.tensorflow.internal.c_api.AbstractTF_Status.throwExceptionIfNotOK(AbstractTF_Status.java:87)
	at ai.djl.tensorflow.engine.javacpp.JavacppUtils.runSession(JavacppUtils.java:192)
	at ai.djl.tensorflow.engine.TfSymbolBlock.forwardInternal(TfSymbolBlock.java:128)
	at ai.djl.nn.AbstractBlock.forward(AbstractBlock.java:126)
	at ai.djl.nn.Block.forward(Block.java:122)
	at ai.djl.inference.Predictor.predictInternal(Predictor.java:130)
	at ai.djl.inference.Predictor.batchPredict(Predictor.java:157)
	... 3 more
```

## How to Reproduce?

```
   @Override
        public NDList processInput(TranslatorContext ctx, String input) {
        	 List<String> tokens = new ArrayList<>();
        	 tokens.add("[CLS]");
        	 tokens.addAll(tokenizer.tokenize(input));
        	 while (tokens.size() < maxSequenceLength) 
        		 tokens.add("[PAD]");
              int[] indices = tokens.stream().mapToInt(l-> (int)vocabulary.getIndex(l)).toArray();
              int[] attentionMask = new int[tokens.size()];
              Arrays.fill(attentionMask, 1);
              NDManager manager = ctx.getNDManager();
              NDArray indicesArray = manager.create(indices);
              NDArray attentionMaskArray = manager.create(attentionMask);
              return new NDList(indicesArray, attentionMaskArray);
```
        	

### Steps to reproduce
(Paste the commands you ran that produced the error.)

Convert distillbert pytorch model to tensorflow . 

## What have you tried to solve it?

1. Remove cls and padding , input of 5 tokens works perfectly.

## Environment Info

Please run the command `./gradlew debugEnv` from the root directory of DJL (if necessary, clone DJL first). It will output information about your system, environment, and installation that can help us debug your issue. Paste the output of the command below:

```
PASTE OUTPUT HERE
```

**Comment by frankfliu** [comment]
@pmg1991  Can you share a complete project with minimal reproduce code?

**Comment by pmg1991** [comment]
@frankfliu will this work?

**Comment by pmg1991** [comment]
@frankfliu  Please download converted model from here https://drive.google.com/file/d/1yM3LEfsZjl7CiIx4gxv36mb_a_nwpMmU/view?usp=sharing

**Comment by frankfliu** [comment]
Feel free to reopen this issue if you have more questions
